src/platforms/java/migration.mdx

```diff
@@ -109,3 +109,55 @@ _New_:
 ```java
 Sentry.setExtra("extra", "thing");
 ```
+
+### Property Name Changes
+
+Following properties cannot be set anymore through external configuration and have to be set directly on `SentryOptions` object when initializing Sentry:
+
+- `mdcTags`
+- `extra`
+- `sample.rate`
+- `buffer.dir`
+- `buffer.size`
+- `buffer.flushtime`
+- `buffer.shutdowntimeout`
+- `buffer.gracefulshutdown`
+- `async`
+- `async.shutdowntimeout`
+- `async.gracefulshutdown`
+- `async.queuesize`
+- `async.threads`
+- `async.priority`
+- `compression`
+- `maxmessagelength`
+- `timeout`
+- `factory`
```

Comment:
not all of them are available in the new SDK though, I'd separate them, addind a section for deprecated/removed